.github/workflows/sync-upstream.yaml

```
@@ -0,0 +1,28 @@
+name: sync-upstream
+
+on:
+  schedule:
+    - cron: 0 19 * * * # run at 4 AM JST
+  workflow_dispatch:
+
+jobs:
+  sync-upstream:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Run sync-branches
+        uses: autowarefoundation/autoware-github-actions/sync-branches@tier4/proposal
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          base-branch: tier4/universe
```

Is there the `tier4/universe` branch?

```suggestion
          base-branch: main
```

---

It's a mistake. Thanks!